aws/scripts/cloud-init-cratedb-rpm.tftpl

```diff
@@ -87,6 +87,31 @@ write_files:
     owner: root:root
     path: /etc/sysconfig/crate
     permissions: "0755"
+  - content: |
+      [prometheus-rpm_release]
+      name=prometheus-rpm_release
+      baseurl=https://packagecloud.io/prometheus-rpm/release/el/8/$basearch
+      repo_gpgcheck=0
+      gpgcheck=0
```

Comment:
Is this necessary? would it work with the default of 1?

Reply:
That's how `script.rpm.sh` has generated the repository configuration.

If the OS is `el/7` or later, it skips installing `pygpgme` which is later used for importing the key: 
```shell
  if [[ "$os" = "ol"  ||  "$os" = "el" ]] && [ $(($dist)) \> 7 ]; then
    _skip_pygpgme=1
  else
    _skip_pygpgme=0
  fi
```
`pygpgme` isn't available on Amazon Linux. I tried both `rpm --import https://packagecloud.io/prometheus-rpm/release/gpgkey` and `rpmkeys --import https://packagecloud.io/prometheus-rpm/release/gpgkey`, but with `gpgcheck=0`, `dnf` still complained that `The GPG keys are already installed but they are not correct for this package.` 🤷.

Reply:
Found out there is a cloud-init `yum_repos` module which simplifies the setup a bit. But even using that, setting `gpgcheck: true` leads to an error. I would assume the cloud-init module should take care of importing keys and be able to figure it out? But looks like it doesn't/can't...